Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ however, it has to be formatted properly to pass verification tests.
 ## [Unreleased] - yyyy-mm-dd
 - Fixed Composite binding isn't triggered after ResetDevice() called during Action handler [ISXB-746](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-746)
 - Fixed resource designation for "d_InputControl" icon to address CI failure
+- Changed FastMouse.OnNextUpdate() to pass the current time as a parameter to InputState.Change() improving performance (case ISX-1414)
```

Comment:
Do you know what is the performance impact? 
Should QA look at any performance tests to validate this claim?

Reply:
It will be quite minor I think - it's just reducing a redundant call from scripting to native and back to get the current accurate time.

Reply:
I could change the changelog I suppose:

- Changed FastMouse.OnNextUpdate() to pass the current time as a parameter to InputState.Change() avoiding a redundant scripting to native call to query the time (case ISX-1414).

Reply:
@AlexTyrer I see, thanks for clarifying. Yeah, maybe that sounds better. Fine for me 👍